src/js/components/reviewData/CertificationModal/RevalidateDataModal.jsx

```diff
@@ -0,0 +1,99 @@
+/**
+  * RevalidateDataModal.jsx
+  * Created by Nipun Monga 02/27/17
+  **/
+
+import React from 'react';
+import Modal from 'react-aria-modal';
+import { hashHistory } from 'react-router';
+import * as Icons from '../../SharedComponents/icons/Icons.jsx';
+
+import RevalidateButtons from './RevalidateButtons.jsx';
+import * as ReviewHelper from '../../../helpers/reviewHelper.js';
+
+export default class ReviewDataModal extends React.Component {
+	constructor(props) {
+		super(props);
+
+		this.state = {
+			closeable: true,
+			errorMessage: ""
+		};
+	}
+
+	clickedRevalidateButton(e) {
+        e.preventDefault();
+
+        ReviewHelper.revalidateSubmission(this.props.submissionID)
+            .then(() => {
+                this.closeModal();
+                // Redirect to validateData page
+                hashHistory.push('/validateData/' + this.props.submissionID);
+            })
+            .catch((error) => {
```

Comment:
Something that Bray told me, rather than if-else, store error message into a variable then 
`if(error.httpStatus == 400 || error.httpStatus == 403){
    errorMessage: error.message;
}`
then setState after. That way there is only one setState and reduces logic slightly